pensieve/analysis.py

```diff
@@ -179,7 +185,7 @@ def _calculate_statistics(self, metrics_table: str, period: AnalysisPeriod):
         results = []
 
         for m in self.config.metrics[period]:
-            results.append(m.run(metrics_data).to_dict())
+            results += m.run(metrics_data).to_dict()["data"]
```

Comment:
The version currently deployed doesn't write any statistics data to BigQuery because of this.